doc-surrealdb_versioned_docs/version-2.x/surrealql/statements/define/function.mdx

```diff
@@ -75,6 +80,32 @@ RETURN fn::last_option(1);
 };
 ```
 
+## Permissions
+
+You can set the permissions for a custom function using the `PERMISSIONS` clause. The `PERMISSIONS` clause is mostly used to restrict who can access a function and what data they can access. It can be set to `NONE`, `FULL`, or `WHERE @condition`.
+
+- `FULL`: When Full permissions are granted [record](/docs/surrealdb/2.x/security/authentication#record-users) users have access to the function. This is the default permission when not specified.
+- `NONE`: When this permission is granted, [record](/docs/surrealdb/2.x/security/authentication#record-users) users  have no access to the defined function.
+- `WHERE @condition`: Permissions are granted to the function based on the specified condition.
+
+### Using the `FULL` permission
```

Comment:
should we also include an example for the `NONE` clause?

Reply:
Ah! Added one now